checkov/terraform/checks/resource/tencentcloud/CDBInternetService.py

```diff
@@ -0,0 +1,21 @@
+from checkov.common.models.enums import CheckCategories, CheckResult
+from checkov.terraform.checks.resource.base_resource_value_check import \
+    BaseResourceCheck
+
+
+class CDBInternetService(BaseResourceCheck):
```

Comment:
Not a required change, but `BaseResourceNegativeValueCheck` would make more sense for this check